Newsletters: add 321 (2024-09-20)

harding opened the pull request:
- [x] News items @harding 
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Client/services @bitschmidty 
- [x] Notable merges @Gustavojfe

vostrnad commented on _posts/en/newsletters/2024-09-20-newsletter.md:
```diff
@@ -0,0 +1,87 @@
+  `bdk_chain`, `bdk_electrum`, `bdk_esplora`, and `bdk_bitcoind_rpc`.
+  The `bdk_wallet` crate "is the first version to offer a stable 1.0.0 API."
+
+- [Bitcoin Core 28.0rc1][] is a release candidate for the next major
+  version of the predominant full node implementation.  A [testing
+  guide][bcc testing] is available.
```
[v28.0rc2](https://github.com/bitcoin/bitcoin/releases/tag/v28.0rc2) was tagged yesterday.

Gustavojfe commented:
just added merge summaries

bitschmidty commented on _posts/en/newsletters/2024-09-20-newsletter.md:
maybe

```suggestion
  to Delving Bitcoin a survey of the availability of seed nodes on
```
```suggestion
  to Delving Bitcoin a survey of the availability of seed nodes for
```

bitschmidty commented on _posts/en/newsletters/2024-09-20-newsletter.md:
```suggestion
  implement, even on devices with limited computing resources, like smart
```
```suggestion
  implement even on devices with limited computing resources, like smart
```

bitschmidty commented on _posts/en/newsletters/2024-09-20-newsletter.md:
```suggestion
[delving cluster]: https://delvingbitcoin.org/t/how-to-linearize-your-cluster/303#h-2-finding-high-feerate-subsets-5
```

bitschmidty commented on _posts/en/newsletters/2024-09-20-newsletter.md:
```suggestion
- [Bitcoin Core #30807][] changes the signaling of an [assumeUTXO][topic
```

bitschmidty commented on _posts/en/newsletters/2024-09-20-newsletter.md:
maybe

```suggestion
  `NODE_NETWORK_LIMITED` so that peer nodes don’t request a historical block from it. This
```

bitschmidty commented on _posts/en/newsletters/2024-09-20-newsletter.md:
Might be good to note briefly what quarantining the type means

bitschmidty commented on _posts/en/newsletters/2024-09-20-newsletter.md:
```suggestion
  payment [onion message][topic onion messages]. Sending as an often offline
```

azuchi commented on _posts/en/newsletters/2024-09-20-newsletter.md:
```suggestion
and summarizing notable changes to popular Bitcoin infrastructure
```

harding commented:
Made edits for all feedback; added lede; updated releases/RCs; and reviewed other contributions.  Also updated topic entries for this newsletter and also newsletters 319 and 320, which I ran out of time to do.  Thanks everyone (and apologies for my tardiness)!

bitschmidty commented:
Pushed a small change to the news section. ACK topics from 321, 320, 319. Squashed. Thank you @harding @Gustavojfe @azuchi @vostrnad for contributing this week!
